script/Ciel/Day 13/13_01_02_CIEL13_8.txt

```diff
@@ -167,14 +167,15 @@ I feel like if I don't make myself this clear,
 -- Page 483, Offset 36649.
 -- 「……しょうのない後輩ですね。
 // translators found dead in a ditch, trace amount of 後輩 detected in their systems
+// Were we removing all of these?
 // 困る、仕様がない, 日常、後輩... not even once
 // VA has a sigh before the line starts
 "%{i}Sigh%{/i}... Just what am I supposed to do with you.
 }
 [sha:cc5dd576701a0371e82a8745a2a00c964a78e10e]{
 -- Page 483, Offset 36650.
 -- 聞いているこっちの方が恥ずかしくなるなんて、想定外もいいところです。そんなコトを言われたら、完全降伏するしかないじゃないですか」
-I didn't expect hearing you say all that embarrassing stuff to make %{i}me%{/i} embarrassed, but it's not the worst feeling.
+I didn't expect hearing you say all that embarrassing stuff would make %{i}me%{/i} embarrassed, but it's not the worst feeling.
```

Comment:
Hmm, are you sure about this one? I feel like `expect -> to` makes more sense to me here. I'd use `would` if the `expect` was `think` instead maybe.